config/admin-cli.properties

```diff
@@ -9,13 +9,16 @@
 #
 
 # Config for CLI credentials (at the moment, Controller auth is only supported).
-cli.security.auth.enable=false
-cli.security.auth.credentials.username=admin
-cli.security.auth.credentials.password=1111_aaaa
+cli.controller.connect.channel.auth=false
+cli.controller.connect.channel.tls=false
+cli.controller.connect.credentials.username=admin
+cli.controller.connect.credentials.pwd=1111_aaaa
+cli.controller.connect.delegationToken.signingKey.basis=secret
```

Comment:
What is this used for in CLI?

Reply:
When we run the CLI against the standalone in auth scenario, we use this token to set up a `StreamMetadataStore`.
[In these lines.](https://github.com/anirudhkovuru/pravega/blob/ac8067b0e14bed7dffab81a2c11032d6e3705e0a/cli/admin/src/main/java/io/pravega/cli/admin/controller/ControllerDescribeStreamCommand.java#L69-L82)
We generate an `authHelper` using the token to make the store.

Reply:
I still don't understand it. `signing key` is a server secret, and it is only supposed to be used across the Controller and the Segment Store. Could you please help me understand the intent better?

Reply:
The signing key was removed, so this issue is resolved.